src/js/quizValidation.js

```diff
@@ -0,0 +1,9 @@
+let score = 0;
+const logicAndScore = (questionCounter = 0, answersCounter = 0) => {
+    const dataValidation = shuffledQuizData[questionCounter].answers[answersCounter].correct;
+
+    if (dataValidation === true) {
+        score++;
+    }
+    return score;
+};
```

Comment:
gdzie jakis export i jak to w ogole z zalozenia powinno dzialac? troche nie kumam